docs/Examples.md

```diff
@@ -87,6 +87,183 @@ $graphServiceClient = GraphServiceClient::createWithRequestAdapter($requestAdapt
 
 ```
 
+## Access token management
+Using the `TokenRequestContext`, an instance of the `GraphServiceClient` requests access tokens and refresh tokens.
+The tokens are stored by default in an in-memory cache so that future requests using the same instance of the `GraphServiceClient` can re-use the previously acquired tokens.
+
+For scenarios where an application requires a signed-in user, retaining the same
```

Comment:
What is "a user session"? Multiple page visits, for instance? Or multiple calls to graph as part of a single page load?

Reply:
Updated this section. Is it clearer?